Cargo.toml

```diff
@@ -15,7 +15,9 @@ std = []
 
 # Supports const trait implementation through const_convert and const_trait_impl. Support for those was removed
 # from more recent Rust nightlies, so this feature requires an older Rust compiler
-# (2023-04-20 is broken, 2022-11-23 works. The exact day is somewhere inbetween)
+# (2023-04-20 is broken, 2022-11-23 works. The exact day is somewhere inbetween).
+# As of 12/2/2024, this also uses inline_const. This has been stabilized but is required for current code to work
+# on the old compiler.
 const_convert_and_const_trait_impl = []
```

Comment:
@danlehmann With 1.3.0 being incompatible to previous versions anyway and `inline_const` having been stabilized in Rust 1.79.0, may it be a good time to raise the MSRV to 1.79 and just remove this optional feature?

Reply:
This feature is still used by https://crates.io/crates/bilge (which itself puts it behind an optional feature). While I really dislike the code duplication, I don't want to break them. I think the right time to remove the feature is when const_convert/const_trait become finalized.

In terms of breakage, I think 1.3.0 doesn't break much. I'd like to keep it at a minimum.